src/cli.js

```diff
@@ -73,7 +73,7 @@ yargs
       target: {
         demand: false,
         describe: 'Code generation target language',
-        choices: ['swift', 'json', 'ts', 'typescript'],
+        choices: ['swift', 'json', 'ts', 'typescript', 'flow', 'flowtype'],
```

Comment:
Do we need both `flow` and `flowtype` here? I would say we just pick just one (`flow`?) to avoid confusion.